scripts/rewrite/resources/prompts/post-to-canny.md

```
@@ -0,0 +1,71 @@
+You are writing a **changelog entry** that will be posted to our Canny
+board (non-technical users + stakeholders read this) for PR
+#{{PR_NUM}} which just merged into `ralph-looped`.
+
+The PR implements ticket **{{TICKET_ID}}** (issue #{{ISSUE_NUM}}).
+
+---
+
+## PR context
+
+**Title:** {{PR_TITLE}}
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)